### PR TITLE
Updated (C) from 2016 to 2017

### PR DESCRIPTION
**Pull request by ObloxCC**

### CODE REVIEW

**Comment by dan200** [comment]
Please remove all the changes except for the date change

**Comment by Wilma456** [comment]
The text "Do not distribute without permission" should be removed, because Computercraft is now under a free license and everybody has the permission to distribute.

**Comment by apemanzilla** [comment]
The text `Do not distribute` still holds, the new license is what gives you permission. If the license changes again distribution may not be allowed.

**Comment by dan200** [comment]
The license is the permission! Agree it could be rewritten though

**Comment by Wilma456** [comment]
We can change the text back, if dann200 change the License.  In the actual license stands:

> The user is allowed to redistribute this mod partially, in totality, or 
included in a distribution.

**Comment by apemanzilla** [comment]
@Wilma456 yes, the license gives everyone the permission to distribute as stated in the copyright notices in the files. It's fine the way it is.

**Comment by ObloxCC** [comment]
> Please remove all the changes except for the date change

None of the other 'changes' are changes besides the date, Atom/GitHub has some problems where it thinks a line was edited when there was no change to that line.

**Comment by dan200** [comment]
There are numerous whitespace changes

**Comment by Wojbie** [comment]
Yea.. There is more then one Whitespace change mate (like 50?). And you reversed only one.

**Comment by SquidDev** [comment]
@ObloxCC Go into your Atom settings and disable "remove trailing whitespace". This'll stop this happening in the future. I'd also recommend turning on "show invisibles" (I think that's what it's called, been a while since I used Atom).

@dan200 If at all possible, could you do a "reformat" commit, where you trim all whitespace, normalise spaces/tabs in the Lua files and standardise Java formatting. It's rather frustrating as each file is formatted slightly differently.

**Comment by ObloxCC** [comment]
> Yea.. There is more then one Whitespace change mate (like 50?). And you reversed only one.

I was ensuring that it would correctly fix the whitespaces without the previous error (editor deleting them)

**Comment by ObloxCC** [comment]
All whitespaces should be fixed

**Comment by SquidDev** [comment]
There are still a couple of files with some whitespace changes:

- src/main/java/dan200/computercraft/client/gui/FixedWidthFontRenderer.java
- src/main/java/dan200/computercraft/client/gui/GuiPrinter.java
- src/main/java/dan200/computercraft/client/gui/GuiPrintout.java
- src/main/java/dan200/computercraft/client/gui/GuiTurtle.java
- src/main/java/dan200/computercraft/client/gui/widgets/WidgetTerminal.java
- src/main/java/dan200/computercraft/client/proxy/CCTurtleProxyClient.java
- src/main/java/dan200/computercraft/shared/computer/blocks/ComputerState.java
- src/main/java/dan200/computercraft/shared/peripheral/common/TilePeripheralBase.java

You can tell this as the number of files changed (231) is slightly less than the number of deletions/additions (259/260). Ideally all three numbers should be the same.

It *might* be slightly easier just do undo all your commits and do a "fresh replace in directory", this time with whitespace trimming disabled.

File: src/main/java/dan200/computercraft/client/gui/GuiPrinter.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 public class GuiPrinter extends GuiContainer
 {
     private static final ResourceLocation background = new ResourceLocation( "computercraft", "textures/gui/printer.png" );
-    
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/gui/GuiPrinter.java]: In this file.

File: src/main/java/dan200/computercraft/client/gui/GuiPrintout.java [hunk 2]
```diff
@@ -21,10 +21,10 @@
 public class GuiPrintout extends GuiContainer
 {
     private static final ResourceLocation background = new ResourceLocation( "computercraft", "textures/gui/printout.png" );
-    
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/gui/GuiPrintout.java]: In this file.

File: src/main/java/dan200/computercraft/client/gui/FixedWidthFontRenderer.java [hunk 3]
```diff
@@ -165,7 +165,7 @@ public void drawString( TextBuffer s, int x, int y, TextBuffer textColour, TextB
         {
             // Bind the font texture
             m_textureManager.bindTexture( font );
-            
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/gui/FixedWidthFontRenderer.java]: Just pointing out some whitespace changes left in files.
In this file.

File: src/main/java/dan200/computercraft/client/gui/GuiTurtle.java [hunk 4]
```diff
@@ -29,15 +29,15 @@
 {
     private static final ResourceLocation background = new ResourceLocation( "computercraft", "textures/gui/turtle.png" );
     private static final ResourceLocation backgroundAdvanced = new ResourceLocation( "computercraft", "textures/gui/turtle_advanced.png" );
-    
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/gui/GuiTurtle.java]: In here.

File: src/main/java/dan200/computercraft/client/proxy/CCTurtleProxyClient.java [hunk 5]
```diff
@@ -44,8 +44,8 @@
 public class CCTurtleProxyClient extends CCTurtleProxyCommon
 {
     // IComputerCraftProxy implementation
-    
-    @Override        
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/proxy/CCTurtleProxyClient.java]: In here.

File: src/main/java/dan200/computercraft/client/gui/widgets/WidgetTerminal.java [hunk 6]
```diff
@@ -292,10 +292,10 @@ else if( wheelChange > 0 )
 
     @Override
     public void update()
-    {                
+    {
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/client/gui/widgets/WidgetTerminal.java]: In here.

File: src/main/java/dan200/computercraft/shared/peripheral/common/TilePeripheralBase.java [hunk 7]
```diff
@@ -111,7 +111,7 @@ public synchronized int getAnim()
     {
         return m_anim;
     }
-    
+
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/shared/peripheral/common/TilePeripheralBase.java]: And finally this one.
Only 8 files left to clean!

File: src/main/java/dan200/computercraft/shared/computer/blocks/ComputerState.java [hunk 8]
```diff
@@ -36,4 +36,3 @@ public String toString()
         return getName();
     }
 }
-
```
Comment (Wojbie) [inline src/main/java/dan200/computercraft/shared/computer/blocks/ComputerState.java]: This file

**Comment by ObloxCC** [comment]
And done.

**Comment by dan200** [comment]
Good work!

**Pull request merged**